docs/modules/ROOT/pages/push.adoc

```diff
@@ -0,0 +1,94 @@
+== AeroGear JavaScript Unifiedpush SDK
```

Comment:
`UnifiedPushServer`

Reply:
Some of the instructions bellow are strictly Cordova related. IMHO it's best to just call that Cordova SDK. If not we should provide documentation for push registration only and possibly have separate page for plugin. WDYT?

Reply:
You mean, it would be possible to use @aerogear/push in a react-native app, but then you would obviously need to use a different push implementation? Yes i guess we can separate those the docs with regards to this.

Reply:
That's tough one. We can introduce here to the topic and create Cordova chapter. We do not need separate file. Then if React Native will be there we can just add another chapter.